packages/admin-rte/src/core/BlockElement.tsx

```diff
@@ -0,0 +1,95 @@
+import { makeStyles, Typography, TypographyProps } from "@material-ui/core";
+import { Theme } from "@material-ui/core/styles";
+import * as React from "react";
+
+import { SupportedThings } from "./Rte";
+
+// Only block-types used in block-type-map should be styled
+type StylableBlockTypes = Extract<
+    SupportedThings,
+    "header-one" | "header-two" | "header-three" | "header-four" | "header-five" | "header-six" | "blockquote" | "unordered-list" | "ordered-list"
+>;
+
+export type CometAdminRteBlockElementClassKeys = StylableBlockTypes | "root";
+
+interface Props extends TypographyProps {
+    type?: StylableBlockTypes;
+    component?: React.ElementType;
+}
+
+export const BlockElement = ({ type, ...restProps }: Props) => {
+    const classes = useStyles();
+    return <Typography classes={{ root: `${classes.root} ${type ? classes[type] : ""}` }} {...restProps} />;
+};
+
+const useStyles = makeStyles<Theme, {}, CometAdminRteBlockElementClassKeys>(
+    ({ palette }) => ({
+        root: {
+            fontSize: 16,
+            lineHeight: "20px",
+            fontWeight: 300,
+            color: palette.grey[800],
+            marginBottom: 10,
+        },
+        "header-one": {
+            fontSize: 40,
```

Comment:
isn't that still too large?

Reply:
@nsams yes, now I think it looks good, even if there isn't a lot of space for the RTE.